app/controllers/account/tenant_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Account::TenantController < ApplicationController
+  before_action :authenticate_user!
+
+  def show
+    @tenant = current_tenant
+  end
+
+  def edit
+    @tenant = current_tenant
+  end
+
+  def update
+    @tenant = current_tenant
+    if @tenant.update_attributes(tenant_params)
```

Comment:
it's better to use `ActiveRecord::Persistence#update` method to update record within controller,
a bit safer and doesn't allow mass-assignment

Reply:
Done.